components/form-builder/panel/ElementPanel.tsx

```diff
@@ -438,13 +442,13 @@ export const ElementWrapper = ({ item }: { item: ElementTypeWithIndex }) => {
           </ModalButton>
         )}
       >
-        {modals[item.index] && (
+        {!isRichText && modals[item.index] ? (
```

Comment:
The previous usage of `&&` is preferred, as it omits the unecessary ` : null` as the other part of the ternary.

Reply:
Update in https://github.com/cds-snc/platform-forms-client/pull/1056/commits/fb321043948986e1562815a208bc4f77dc2ad1c5